src/test/java/teammates/test/cases/BaseTestCase.java

```diff
@@ -113,6 +113,13 @@ protected static String getPopulatedErrorMessage(String messageTemplate, String
                                      null, new Object[] { messageTemplate, userInput, fieldName, errorReason, maxLength });
     }
 
+    protected static String getPopulatedEmptyStringErrorMessage(String messageTemplate, String fieldName, int maxLength)
+            throws ReflectiveOperationException {
+        return (String) invokeMethod(FieldValidator.class, "getPopulatedEmptyStringErrorMessage",
```

Comment:
Can call the method directly, instead of using reflection.
`return FieldValidator.getPopulatedEmptyStringErrorMessage(...);`